packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerItem.tsx

```diff
@@ -163,30 +183,58 @@ const StyledSpacer = styled.span`
   flex-grow: 1;
 `;
 
-const StyledRightOptionsContainer = styled.div<{
-  isMobile: boolean;
-  active: boolean;
-}>`
-  margin-left: auto;
-  visibility: ${({ isMobile, active }) =>
-    isMobile || active ? 'visible' : 'hidden'};
+const StyledIcon = styled.div`
+  flex-shrink: 0;
+  flex-grow: 0;
+  margin-right: 8px;
```

Comment:
use theme for `margin-right: ${({theme})} => ...`

Reply:
Ah! I missed that ;(